crates/common/src/file.rs

```diff
@@ -0,0 +1,28 @@
+use std::{fs::File, path::Path};
+
+use reqwest::blocking as reqwest;
+use zip::CompressionMethod;
+use zip_extensions::ZipWriterExtensions;
+
+pub fn download_file<S: Into<String>, P: AsRef<Path>>(source: S, dest: P) -> anyhow::Result<()> {
+    let mut response = reqwest::get(source.into())?;
```

Comment:
`source` probably makes more sense to be `AsRef<str>` since we don't need an owned String here, reqwest::get will copy it anyway